### PR TITLE
Add support of router script

### PR DESCRIPTION
**Pull request by maidmaid**
cf #98

### CODE REVIEW

**Comment by dunglas** [comment]
The  of the method's parameters starts to be huge. Maybe should be change the signature like this:


```php
public static function startWebServer(array $options = [])
protected static function createPantherClient(array $options = [], kernelOptions = [])
```

The other benefit is that introducing new options will not be a BC break anymore.

**Comment by maidmaid** [comment]
Good idea, but can we do it now or it is a BC ?

**Comment by dunglas** [comment]
We can do it now, we're still in a 0.x version so BC are allowed (and expected).

**Comment by maidmaid** [comment]
In this PR ? (I asked that because you approved it)

**Comment by dunglas** [comment]
Yes it can be done in this one (because the tests are already broken because of this). Another one is fine too, as you prefer :)

File: src/PantherTestCaseTrait.php [hunk 1]
```diff
@@ -56,6 +56,16 @@
      */
     protected static $pantherClient;
 
+    /**
+     * @var array
+     */
+    protected static $defaultOptions = [
```
[the remark this answers is not in the document]
Comment (maidmaid) [reply src/PantherTestCaseTrait.php]: Not in a trait...

File: src/ProcessManager/WebServerManager.php [hunk 2]
```diff
@@ -34,7 +34,7 @@
     /**
      * @throws \RuntimeException
      */
-    public function __construct(string $documentRoot, string $hostname, int $port)
+    public function __construct(string $documentRoot, string $hostname, int $port, string $router = '')
```
Comment (maidmaid) [inline src/ProcessManager/WebServerManager.php]: I changed `$router = null` to `$router = ''` (better support into the Process).

**Comment by maidmaid** [comment]
Could you review again please ?

**Comment by dunglas** [comment]
Thanks @maidmaid!

**Pull request merged**